olm-catalog/serverless-operator/project.yaml

```diff
@@ -69,3 +69,5 @@ upgrade_sequence:
     source: redhat-operators
   - csv: serverless-operator.v1.28.0
     source: serverless-operator
+  - csv: serverless-operator.v1.29.0
```

Comment:
we add new entries now?

Reply:
Not sure, how/what would update it otherwise?

Reply:
Came from https://github.com/openshift-knative/serverless-operator/pull/1962. @mgencur is that change here ok?

Reply:
Yeah. This addition looks ok. The kitchensink upgrade tests run periodically. At some point we can start removing items from the beginning of the list. But for now, I'd leave serverless-operator.v1.25.0 there because it uses the old API for operator CRDs.